indirect.h

```diff
@@ -465,10 +465,17 @@ concept is_hashable = requires(T t) { std::hash<T>{}(t); };
 template <typename Value>
 indirect(Value) -> indirect<Value>;
 
-template <typename Alloc, typename Value>
+template <typename Alloc, typename Value,
+          typename std::enable_if_t<!is_indirect_v<Value>, int> = 0>
```

Comment:
```suggestion
template <typename Alloc, typename Value>
```
The `enable_if` isn't needed, since the deduction guide on line 475 is more specific. (Please correct me if I'm wrong; I didn't test this.)